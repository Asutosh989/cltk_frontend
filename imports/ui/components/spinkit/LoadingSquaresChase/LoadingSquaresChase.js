export default class LoadingSquaresChase extends React.Component {
	render() {
		return (
			<div className="loading-spinner loading-spinner-squares-chase">
				<div className="cube1"></div>
				<div className="cube2"></div>
			</div>
		);
	}
}
