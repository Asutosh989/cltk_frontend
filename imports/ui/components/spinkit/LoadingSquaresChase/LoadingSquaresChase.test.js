
import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import LoadingSquaresChase from './LoadingSquaresChase';

describe('LoadingSquaresChase', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<LoadingSquaresChase />
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
