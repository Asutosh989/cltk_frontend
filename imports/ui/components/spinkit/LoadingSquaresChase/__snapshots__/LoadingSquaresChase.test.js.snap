// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSquaresChase renders correctly 1`] = `
<div
  className="loading-spinner loading-spinner-squares-chase"
>
  <div
    className="cube1"
  />
  <div
    className="cube2"
  />
</div>
`;
